code/modules/butchery/butchery_data_livestock.dm

```diff
@@ -24,6 +24,17 @@
 	skin_material = /decl/material/solid/organic/skin/goat
 	skin_amount   = 8
 
+/decl/butchery_data/animal/ruminant/deer
+	meat_name     = "venison"
+	meat_type     = /obj/item/chems/food/butchery/meat/
```

Comment:
either you meant for this to be a nonexistent subtype, or you should remove the trailing `/`.